packages/proxy-agent/package.json

```diff
@@ -36,7 +36,7 @@
     "debug": "^4.3.4",
     "http-proxy-agent": "^7.0.1",
     "https-proxy-agent": "^7.0.5",
-    "lru-cache": "^7.14.1",
+    "lru-cache": "^11.0.0",
```

Comment:
`lru-cache@11` supports Node `20 || >= 22`: https://github.com/isaacs/node-lru-cache/blob/65c9971e3fef123ff0f17d67add53b2d99b461b0/package.json#L70

So updating `lru-cache` would require a major version bump. But `dispose` is available in `v7` (https://github.com/isaacs/node-lru-cache/commit/7a6f529e2e7c1bc3c81f3ee996267ef2006de492) so I think its better to revert the dep update and save it for the next major.